R/powerpoint_presentation.R

```diff
@@ -54,6 +70,27 @@ powerpoint_presentation <- function(
       args = args
     ),
     keep_md = keep_md,
-    df_print = df_print
+    df_print = df_print,
+    intermediates_generator = intermediates_generator
   )
 }
+
+powerpoint_intermediates_generator <- function(saved_files_dir,
+                                               original_input,
+                                               encoding, intermediates_dir) {
+
+  # copy all intermediates (pandoc will need to bundle them in the Powerpoint)
+  intermediates <- copy_render_intermediates(original_input, encoding,
+                                             intermediates_dir, FALSE)
+
+  # we need figures from the supporting files dir to be available during
+  # render as well; if we have a files directory, copy its contents
+  if (!is.null(saved_files_dir) && dir_exists(saved_files_dir)) {
```

Comment:
It seems `saved_files_dir` is always `NULL`? I don't see where it can be assigned a different value.